lstchain/mc/sensitivity.py

```diff
@@ -420,21 +432,20 @@ def find_best_cuts_sensitivity(simtelfile_gammas, simtelfile_protons,
 
     total_rate_proton = np.sum(rate_weighted_p)
     total_rate_gamma = np.sum(rate_weighted_g)
-    print("Total rate triggered proton [Hz]", total_rate_proton)
-    print("Total rate triggered gamma [Hz]", total_rate_gamma)
+    print("Total rate triggered proton {:.3f} Hz".format(total_rate_proton))
+    print("Total rate triggered gamma  {:.3f} Hz".format(total_rate_gamma))
```

Comment:
all these printings should be substituted by loggings, but let's make a full refactor of the code in a different PR...

Reply:
ah ok thanks, I will keep it in mind